Dockerfile

```diff
@@ -55,7 +69,21 @@ RUN pip uninstall -y transformer-engine && \
         git checkout FETCH_HEAD; \
     fi && \
     git submodule init && git submodule update && \
-    NVTE_FRAMEWORK=pytorch NVTE_WITH_USERBUFFERS=1 MPI_HOME=/usr/local/mpi pip install .
+    NVTE_FRAMEWORK=pytorch NVTE_WITH_USERBUFFERS=1 MPI_HOME=/usr/local/mpi pip wheel .
+
+FROM ${TRT_PROXY_IMAGE} as trt-llm-proxy
+FROM ${TE_PROXY_IMAGE} as te-proxy
+
+# Final image
+FROM ${BASE_IMAGE} AS final
+WORKDIR /opt
+# needed in case git complains that it can't detect a valid email, this email is fake but works
+RUN git config --global user.email "worker@nvidia.com"
```

Comment:
I think since we're fixing things, we should probably unset this at the very end so that folks don't get surprised their committing code as `worker@nvidia.com`